wiki/markdown/docker.md

```diff
@@ -0,0 +1,61 @@
+# Docker
+
+In Eiffel-Intelligence source code a Dockerfile is provided which helps the developer or user to build the local Eiffel-Intellegence source code changes to a Docker image.
+With the Docker image user can tryout the Eiffel-Intelligence on a Docker Host or in a Kubernetes cluster.
+
+## Requirements
+- Docker 
+
+
+  Linux: https://docs.docker.com/install/linux/docker-ce/ubuntu/
+
+  
+  Windows: https://docs.docker.com/docker-for-windows/install/
+
+## Follow these step to build the Docker image.
+
+1. Build the Eiffel-intelligence war file: 
+`mvn package -DskipTests` 
+
+
+This will produce a war file in the "target" folder.
+2. Build the Docker image with the war file that was produced from previous step: 
```

Comment:
Have number 2 start on a new line for more readability

Reply:
Fixed.